### PR TITLE
Issue 671

### PR DESCRIPTION
**Pull request by dafuga**
closes issue #671 

** Also fixed an issue that was preventing me from importing new watch wallets and linted a few files.

### CODE REVIEW

File: app/shared/components/Tools/Table/Row/Wallet.js [hunk 1]
```diff
@@ -7,10 +7,9 @@ import { Button, Dropdown, Header, Icon, Label, Popup, Table } from 'semantic-ui
 
 import GlobalAccountEdit from '../../../../containers/Global/Account/Edit';
 import GlobalButtonElevate from '../../../../containers/Global/Button/Elevate';
-import GlobalFragmentAuthorization from '../../../Global/Fragment/Authorization';
-import GlobalFragmentBlockchain from '../../../Global/Fragment/Blockchain';
-import GlobalFragmentChainLogo from '../../../Global/Fragment/ChainLogo';
-import GlobalFragmentWalletType from '../../../Global/Fragment/WalletType';
+import FragmentAuthorization from '../../../Global/Fragment/Authorization';
```
Comment (aaroncox) [inline app/shared/components/Tools/Table/Row/Wallet.js]: Why did these get renamed?

File: app/shared/components/Tools/Table/Row/Wallet.js [hunk 2]
```diff
@@ -43,12 +42,18 @@ class ToolsTableRowWallet extends Component<Props> {
...
-    if (walletCount === 1) {
-      actions.changeModule('');
+    if (wallets.length === 1) {
+      return actions.changeModule('');
+    }
+
+    if (settings.account === account) {
+      const otherWallet = wallets.find(wallet => wallet.account !== account);
+
+      actions.useWallet(settings.chainId, otherWallet.account, authorization);
```
Comment (aaroncox) [inline app/shared/components/Tools/Table/Row/Wallet.js]: Should the `authorization` piece here be `otherWallet.authorization`?

File: app/shared/components/Tools/Wallets.js [hunk 3]
```diff
@@ -4,14 +4,11 @@ import { translate } from 'react-i18next';
 
 import { Grid, Header, Segment, Table } from 'semantic-ui-react';
 
-import GlobalButtonAccountImport from '../Global/Button/Account/Import';
+import GlobalAccountImport from '../Global/Button/Account/Import';
```
Comment (aaroncox) [inline app/shared/components/Tools/Wallets.js]: Also why was this renamed?

File: app/shared/components/Tools/Table/Row/Wallet.js (hunk 1 above)
Comment (aaroncox) [reply app/shared/components/Tools/Table/Row/Wallet.js]: Was it the error stating something about how the export defaults shouldn't be named the same, the linting error?

Comment (dafuga) [reply app/shared/components/Tools/Table/Row/Wallet.js]: Yeah if I remember correctly, it was a lint error stating that that variable here can't be the same as one of the exportable functions.

File: app/shared/components/Tools/Wallets.js (hunk 3 above)
Comment (dafuga) [reply app/shared/components/Tools/Wallets.js]: Same reason as above I think :D

File: app/shared/components/Tools/Table/Row/Wallet.js (hunk 2 above)
Comment (dafuga) [reply app/shared/components/Tools/Table/Row/Wallet.js]: Yes it should! Nice catch!

File: app/shared/components/Tools/Table/Row/Wallet.js (hunk 1 above)
Comment (aaroncox) [reply app/shared/components/Tools/Table/Row/Wallet.js]: I think I'm going to disable that warning. I'd honestly I'd rather have them be the same so you can look them up easier.
